classes/utils/Biome.py

```diff
@@ -0,0 +1,137 @@
+from enum import Enum
+
+
+class Biome(Enum):
+
+    def __init__(self, id, string):
+        self.id = id
+        self.string = string
+
+    def id_to_biome(id):
+        for biome in Biome:
+            if biome.id == id:
+                return biome
+        return None
+
+    def name_to_biome(name):
+        for biome in Biome:
+            if biome.name == name:
+                return biome
+        return None
+
+    def string_to_biome(string):
+        for biome in Biome:
+            if biome.biome == string:
+                return biome
+        return None
+
+    OCEAN = (0, 'ocean')
+    PLAINS = (1, 'plains')
+    DESERT = (2, 'desert')
+    MOUNTAINS = (3, 'mountains')
+    EXTREME_HILLS = MOUNTAINS
+    FOREST = (4, 'forest')
+    TAIGA = (5, 'taiga')
+    SWAMPLAND = (6, 'swamp')
+    RIVER = (7, 'river')
+    HELL = (8, 'nether_wastes')
+    NETHER = HELL
+    SKY = (9, 'the_end')
+    END = SKY
+    FROZEN_OCEAN = (10, 'frozen_ocean')
+    FROZEN_RIVER = (11, 'frozen_river')
+    ICE_PLAINS = (12, 'snowy_tundra')
+    TUNDRA = ICE_PLAINS
+    ICE_MOUNTAINS = (13, 'snowy_mountains')
+    MUSHROOM_ISLAND = (14, 'mushroom_fields')
+    MUSHROOM_SHORE = (15, 'mushroom_field_shore')
+    BEACH = (16, 'beach')
+    DESERT_HILLS = (17, 'desert_hills')
+    FOREST_HILLS = (18, 'wooded_hills')
+    TAIGA_HILLS = (19, 'taiga_hills')
+    EXTREME_HILLS_EDGE = (20, 'mountain_edge')
+    MOUNTAIN_EDGE = EXTREME_HILLS_EDGE
+    JUNGLE = (21, 'jungle')
+    JUNGLE_HILLS = (22, 'jungle_hills')
+    JUNGLE_EDGE = (23, 'jungle_edge')
+    DEEP_OCEAN = (24, 'deep_ocean')
+    STONE_BEACH = (25, 'stone_shore')
+    COLD_BEACH = (26, 'snowy_beach')
+    SNOWY_BEACH = COLD_BEACH
+    BIRCH_FOREST = (27, 'birch_forest')
+    BIRCH_FOREST_HILLS = (28, 'birch_forest_hills')
+    ROOFED_FOREST = (29, 'dark_forest')
+    DARK_FOREST = ROOFED_FOREST
+    COLD_TAIGA = (30, 'snowy_taiga')
+    COLD_TAIGA_HILLS = (31, 'snowy_taiga_hills')
+    MEGA_TAIGA = (32, 'giant_tree_taiga')
+    MEGA_TAIGA_HILLS = (33, 'giant_tree_taiga_hills')
+    EXTREME_HILLS_PLUS = (34, 'wooded_mountains')
+    SAVANNA = (35, 'savanna')
+    SAVANNA_PLATEAU = (36, 'savanna_plateau')
+    MESA = (37, 'badlands')
+    BADLANDS = MESA
+    MESA_PLATEAU_F = (38, 'wooded_badlands_plateau')
+    WOODED_BADLANDS_PLATEAU = MESA_PLATEAU_F
+    MESA_PLATEAU = (39, 'MesaPlateau')
```

Comment:
Invalid entry; expceted badlands_plateau instead of MesaPlateau.
```suggestion
    MESA_PLATEAU = (39, 'badlands_plateau')
```